### ISSUE TITLE
Correctness Issues. 

### ISSUE THREAD

**Issue by tongche** [shekkizh/WassersteinGAN.tensorflow#10]
Hi. In this implementation, is batch normalization used in the discriminator? I think in WGAN paper, it is mentioned that you should not use BN in discriminators, right?

In discriminators it seems that the biases term in all convolutional layers are set fixed, and to zero. I don't think this is mentioned in the paper either.

**Comment by shekkizh** [comment]
The authors by discarding BN were trying to prove that WGAN formulation is much more stable - You are welcome to comment it out.

I don't remember whether it was mentioned in the paper but the code that was released with the model did have the bias term removed.
